templates/layout.html

```diff
@@ -5,8 +5,463 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 
     <link rel="stylesheet" type="text/css" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
-
+	<link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.0.3/css/font-awesome.css">
     <title>{% block title %}To-Do App{% endblock %}</title>
+	<style>
```

Comment:
Many of the styles taken from https://bbbootstrap.com/snippets/full-screen/awesome-todo-list-template-25095891 here are redundant and could be cut out.